pkg/parser/common_test.go

```diff
@@ -83,11 +83,7 @@ func (t *testParserSuite) TestParser(c *C) {
 	}
 
 	unsupportedSQLs := []string{
-		"alter table bar ADD FULLTEXT INDEX `fulltext` (`name`) WITH PARSER ngram",
```

Comment:
remove two supported cases handy?

Reply:
YES, because parser also updated when executing
```
GO111MODULE=on go get -u github.com/siddontang/go-mysql@f52d30c9fcb7824f4b6f38e3e4ed1037a1a05acd
```